molo/polls/views.py

```diff
@@ -25,7 +27,11 @@ class DetailView(generic.DetailView):
 
 def poll_results(request, poll_id):
     question = get_object_or_404(Question, pk=poll_id)
-    choices = list(question.choices())
+    page = question.get_main_language_page()
+    qs = Choice.objects.live().child_of(page).filter(
+        languages__language__is_main_language=True)
+    locale = get_locale_code(get_language_from_request(request))
+    choices = [a.get_translation_for(locale) or a for a in qs]
     total_votes = sum(c.votes for c in choices)
```

Comment:
I think this should be based on `qs` and not choices since you want to count the votes against the main language only. Also, just make sure your tests cover different users voting on the same question in different translations